spec/api/user_spec.rb

```diff
@@ -168,13 +178,23 @@ def thread_result(user_id, params)
         last_response.body.should == "{}"
       end
 
-      it "only returns threads with activity from the specified user"  do
-        @comments["t3 c4"].author = @users["u100"]
-        @comments["t3 c4"].save!
-        rs = thread_result 100, course_id: "xyz"
-        rs.length.should == 2
-        check_thread_result_json(@users["u100"], @threads["t3"], rs[0])
-        check_thread_result_json(@users["u100"], @threads["t0"], rs[1])
+      context 'with standalone thread' do
+        before(:each) do
+          make_standalone_thread_with_comments(@users['u100'], 0)
+          make_standalone_thread_with_comments(@users['u101'], 1)
+        end
+
+        it "only returns threads with non-standalone activity from the specified user"  do
+          @comments["t3 c4"].author = @users["u100"]
+          @comments["t3 c4"].save!
+          @comments["standalone t1 c1"].author = @users["u100"]
+          @comments["standalone t1 c1"].save!
+
+          rs = thread_result 100, course_id: "xyz"
```

Comment:
Not sure what `rs` is here. We generally prefer more descriptive variable names at the cost of slightly longer lines.